tests/babel/plugin-debug-label.test.ts

```diff
@@ -13,28 +14,28 @@ const transform = (
     plugins: [[plugin, { customAtomNames }]],
   })?.code
 
-it('Should add a debugLabel to an atom', () => {
+it('should add a debugLabel to an atom', () => {
```

Comment:
Let's have these changes not related to vitest migration in a separate PR.

Reply:
Sure, [done](https://github.com/pmndrs/jotai/pull/1839/commits/704c6313adb88f8f43c03b7bb337de64ba6185c1)!